frontends/web/src/routes/account/add/add.tsx

```diff
@@ -58,46 +60,35 @@ class ManageAccount extends Component<Props, State> {
     public componentDidMount() {
         backendAPI.getSupportedCoins()
             // TEST with only 1 coin
-            // .then(() => (['btc']))
+            // .then(() => ([{coinCode: 'tbtc' as accountApi.CoinCode, name: 'Bitcoin Testnet', canAddAccount: true}]))
```

Comment:
remove?

Reply:
Yea, but I don't have a btconly device and guess that I'd still need to test this.

Reply:
removed